### PR TITLE
Remove assume role functionality. Rely on Terraform. Fixes #112

### LINKED ISSUE #112: Support passing multiple AWS credentials
In the projects I am currently evaluating atlantis for we have chosen to have seperate aws accounts for production and provide no cross account access as a design choice. This is to reduce the number of people who can accidentally blow up something in prod when they thought it was dev. The people that have access to production are limited as much as possible and those that would have access to a full terraform plan and apply in production are further limited. The reason for this is I have seen more than once someone wipe out something in production and they thought it was dev.

I am not sure how I envision this being done but essentially what we are doing locally at the moment is relying on ENV vars to supply credentials and `awsenv` to safely store and switch between sets of credentials. I imagine that we would want to have a place that per environment, project, etc we could specify the credentials to be used.

### PR DESCRIPTION
**Pull request by lkysow**
Execute `terraform plan` and `terraform apply` with `-var atlantis_user={github user}`.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -308,9 +296,10 @@ func (s *Server) postEvents(w http.ResponseWriter, r *http.Request) {
 
 	// webhook requests can either be application/json or application/x-www-form-urlencoded.
 	// We accept both to make it easier on users that may choose x-www-form-urlencoded by mistake
+	// todo: use go-github's ValidatePayload method if https://github.com/google/go-github/pull/693 is merged
 	if r.Header.Get("Content-Type") == "application/x-www-form-urlencoded" {
 		// GitHub stores the json payload as a form value
-		payloadForm := r.PostFormValue("payload")
+		payloadForm := r.FormValue("payload")
```
Comment (lkysow) [inline server/server.go]: unrelated change. Some webhooks use `GET` requests. To be safe, look at all form values (this will include query strings if specified)

File: terraform/terraform_client.go [hunk 2]
```diff
@@ -52,20 +51,15 @@ func (c *Client) Version() *version.Version {
...
+// the provided args in path.
+func (c *Client) RunCommandWithVersion(log *logging.SimpleLogger, path string, args []string, v *version.Version) (string, error) {
 	tfExecutable := "terraform"
 	// if version is the same as the default, don't need to prepend the version name to the executable
 	if !v.Equal(c.defaultVersion) {
 		tfExecutable = fmt.Sprintf("%s%s", tfExecutable, v.String())
 	}
 	terraformCmd := exec.Command(tfExecutable, args...)
 	terraformCmd.Dir = path
-	if len(envVars) > 0 {
```
Comment (anubhavmishra) [inline terraform/terraform_client.go]: 💃
